PySpice/Spice/HSpice/Server.py

```
@@ -0,0 +1,375 @@
+            input_file = os.path.join(tmp_dir, "input.sp")
+            output_base = os.path.join(tmp_dir, "output")
+
+            netlist_str = str(spice_input)
+            if ".options post" not in netlist_str.lower():
+                netlist_str += "\n.options post=1\n"
+
+            with open(input_file, "w") as f:
+                f.write(netlist_str)
+
+            # Acquire the lock to limit the concurrency
+            lock_fd, slot_idx = acquire_hspice_lock(
```

Ruff (0.15.20)

[warning] Unpacked variable `slot_idx` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)